app/src/main/java/de/danoeh/antennapod/fragment/AddFeedFragment.java

```diff
@@ -59,29 +60,33 @@ public View onCreateView(@NonNull LayoutInflater inflater,
                              @Nullable ViewGroup container,
                              @Nullable Bundle savedInstanceState) {
         super.onCreateView(inflater, container, savedInstanceState);
-        View root = inflater.inflate(R.layout.addfeed, container, false);
+        AddfeedBinding viewBinding;
+
+        viewBinding = AddfeedBinding.inflate(getLayoutInflater());
```

Comment:
Could you please make this one single line? Same for the other ViewBinding

```suggestion
        AddfeedBinding viewBinding = AddfeedBinding.inflate(getLayoutInflater());
```